From 12e7df804ec80799e6ec3f04a1508be026be7b19 Mon Sep 17 00:00:00 2001
From: Thomas Faingnaert <thomas.faingnaert@ugent.be>
Date: Tue, 18 May 2021 17:19:59 +0200
Subject: [PATCH 2/2] Include debug information

---
 makefile.linux_amd64 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/makefile.linux_amd64 b/makefile.linux_amd64
index 4aaad84..80caf9e 100644
--- a/makefile.linux_amd64
+++ b/makefile.linux_amd64
@@ -1,5 +1,5 @@
 
-OPTFLAGS=-O -s
+OPTFLAGS=-O -g
 
 # use "-m32" to have a 32bits executable
 ALLFLAGS=-m32 ${OPTFLAGS} -pipe \
-- 
2.25.1

